portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -11,22 +11,99 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
+
+
+
+
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  int displayNumber = 10;
```

Comment:
Can we make the variable a constant?

Reply:
If the user provides a specific number of comments to be fetched then the value changes. If the field is left empty it uses the value 10. Two paths from here:
1. Store number of comments preference for every user along for every user with other details.
2. Provide a default value in the HTML input field itself.